plugins/snapshots/erofs/erofs.go

```diff
@@ -564,6 +636,24 @@ func (s *snapshotter) Remove(ctx context.Context, key string) (err error) {
 				log.G(ctx).WithError(err).WithField("id", id).Warnf("failed to cleanup upperdir")
 			}
 
+			// Close dm-verity device if it exists for this snapshot
+			// The device should already be unmounted by the container runtime before Remove() is called
+			// If the device is still mounted, dmverity.Close will fail safely and we'll log a debug message
+			if s.enableDmverity && id != "" {
```

Comment:
not quite sure it's clean after mount manager is introduced.

Reply:
I think before you make a progress, make sure what is derek's idea on this since if it's a mount manager improvement, that is fine too.